app/scripts/models/auth_brokers/fx-desktop.js

```diff
@@ -93,7 +93,10 @@ define([
       var channel = this._channel || new FxDesktopChannel();
 
       channel.init({
-        window: this.window
+        window: this.window,
+        // The browser is the origin. Events from the browser have no origin.
```

Comment:
postMessages that originate in Firefox Desktop have an origin of the string `null`. @ncalexan points out that Fx for iOS has an origin of the page itself. It would be nice to have this handled in an abstracted manner.

Relying on the string `null` seems brittle to changes in the Gecko code. @ckarlof, @vladikoff and @rfk - suggestions?

Note, `*` is the same as the current behavior.